Gemfile

```diff
@@ -10,3 +10,4 @@ gem "rails", "~> #{ENV['RAILS_VERSION'] || '7.0'}.0"
 gem "cucumber", "~> #{ENV['CUKES_VERSION'] || '7.0'}"
 gem "cucumber-rails"
 gem 'fabrication', github: 'mathieujobin/fabrication', ref: '923cf6fcefd0566b1d6be7bd2f685b89388f4800'
+gem 'sprockets-rails'
```

Comment:
> `sprockets-rails` is not a dependency of Rails anymore

https://github.com/rails/rails/issues/43793#issuecomment-988216868

Reply:
can it be made optional ? or it has to be present ?

Reply:
Oh I see, you are only adding it to the Gemfile, not the `.gemspec`, that's good.

maybe add it as a `development_dependency` anyway?

Reply:
This gem only needed for `cucumber_test_app` to be executed properly.

Should I add it to the `development_dependency` then?